src/main.c

```diff
@@ -77,6 +77,7 @@ static const struct command {
 	{"crossfade",        0,  1, 0, cmd_crossfade,        "[<seconds>]", "Set and display crossfade settings"},
 	{"current",          0,  0, 0, cmd_current,          "", "Show the currently playing song"},
 	{"del",              0, -1, 1, cmd_del,              "<position>", "Remove a song from the queue"},
+    {"delpart",          1, -1, 0, cmd_partitiondelete,  "<name> ...", "Delete partition(s)"},
```

Comment:
Bad indent - please use tab, not spaces.

Reply:
Sorry, thought I'd caught all those.